pages/common/openssl.md

```diff
@@ -14,6 +14,10 @@
 
 `openssl x509 -req -days {{days}} -in {{filename.csr}} -signkey {{filename.key}} -out {{filename.crt}}`
 
+- Only display a domain's certificate's start and end dates:
```

Comment:
Perhaps "Display the start and end dates for a domain's certificate" would read better?

Reply:
👍 that does read a bit better. Updated, thanks!

Reply:
Should it be expiry date instead of end date ? Maybe that fits better for a cert's context.

Reply:
Yeah, sure! I think that flows better.